woocommerce/woocommerce
# Display return to cart link on mobile devices

### Changes proposed in this Pull Request:

This PR ensures that the `← Return to Cart` link remains visible even when zoomed into the checkout page using the Checkout block or when viewing this page on a mobile device.

Closes #43640.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add a product to the cart.
2. Go to the checkout page, that contains the Checkout block.
3. Zoom into that page, e.g. 500% on Chrome.
4. Scroll down to the `Place Order` button.
5. Verify that the `← Return to Cart` link is visible below the `Place Order` button.
6. Open the page with a mobile device or simulate a mobile view. 
7. Verify that the `← Return to Cart` link is visible below the `Place Order` button, when the page is displayed in mobile mode (screen width <= 476px).

<table>
<tr>
<td valign="top">Before:
<br><br>

![store test_checkout_ (1)](https://github.com/woocommerce/woocommerce/assets/3323310/70be8ab4-4cb8-4ecc-a899-a48cacb38ecc)

</td>
<td valign="top">After:
<br><br>

![store test_checkout_ (2)](https://github.com/woocommerce/woocommerce/assets/3323310/40175153-a31c-4bbc-9389-6511ac1a3ff8)

</td>
</tr>
</table>
